app/components/ui/Modal.module.css

```
@@ -27,6 +27,14 @@
   overscroll-behavior: contain;
 }
 
+.modal.fullScreen {
```

Stylelint (17.2.0)

[error] Expected class selector ".fullScreen" to be kebab-case (selector-class-pattern)

(selector-class-pattern)